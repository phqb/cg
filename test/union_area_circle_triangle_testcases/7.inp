0 0 1
-0.21 0.72
-1.35 -1.41
0.73 -0.06