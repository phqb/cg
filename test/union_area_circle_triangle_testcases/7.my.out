1.2070276717